libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/factories/NotificationCreator.kt

```diff
@@ -443,28 +457,33 @@ class DefaultNotificationCreator(
     }
 
     private suspend fun messagingStyleFromCurrentUser(
-        sessionId: SessionId,
         user: MatrixUser,
         imageLoader: ImageLoader,
         roomName: String,
+        isThread: Boolean,
         roomIsGroup: Boolean
     ): MessagingStyle {
         return MessagingStyle(
             Person.Builder()
                 .setName(user.displayName?.annotateForDebug(50))
                 .setIcon(bitmapLoader.getUserIcon(user.avatarUrl, imageLoader))
-                .setKey(sessionId.value)
+                .setKey(user.userId.value)
                 .build()
         ).also {
-            it.conversationTitle = roomName.takeIf { roomIsGroup }
-            it.isGroupConversation = roomIsGroup
+            it.conversationTitle = if (isThread) {
+                stringProvider.getString(CommonStrings.notification_thread_in_room, roomName)
+            } else {
+                roomName
```

Comment:
Previous code was `roomName.takeIf { roomIsGroup }`. Can you double check this change please?